lib/proto/google/pubsub/v1/pubsub.pb.ex

```diff
@@ -327,6 +327,7 @@ defmodule Google.Pubsub.V1.ReceivedMessage do
   field(:ack_id, 1, type: :string, json_name: "ackId")
   field(:message, 2, type: Google.Pubsub.V1.PubsubMessage)
   field(:delivery_attempt, 3, type: :int32, json_name: "deliveryAttempt")
+  field(:publish_time, 4, type: Google.Protobuf.Timestamp, json_name: "publishTime")
```

Comment:
This is a generated file so shouldn't be edited. `:publish_time` exists on `Google.Pubsub.V1.PubsubMessage`